libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/MapboxNavigationViewApi.kt

```diff
@@ -0,0 +1,73 @@
+package com.mapbox.navigation.dropin
+
+import com.mapbox.maps.MapView
+import com.mapbox.navigation.core.arrival.ArrivalObserver
+import com.mapbox.navigation.core.directions.session.RoutesObserver
+import com.mapbox.navigation.core.trip.session.BannerInstructionsObserver
+import com.mapbox.navigation.core.trip.session.LocationObserver
+import com.mapbox.navigation.core.trip.session.RouteProgressObserver
+import com.mapbox.navigation.core.trip.session.TripSessionStateObserver
+
+interface MapboxNavigationViewApi {
```

Comment:
This should be a sealed interface. We don't want anyone else extending it.

Reply:
Done but because of the restrictions for sealed interfaces I had to add an implementing class. It's internal so is transparent to developers as well as us.